packages/components/src/Text/Text.spec.tsx

```diff
@@ -1,8 +1,8 @@
 import * as TextStoryFile from "./Text.stories";
 
-import { render, screen } from "@testing-library/react";
+import React, { useEffect } from "react";
```

Comment:
huh, weird that this isn't an error 🤔

Reply:
Oops! Yeah, it woulda been nice if that had come up as an error; I didn't notice it.

Reply:
Fixed here: https://github.com/chanzuckerberg/edu-design-system/pull/454